src/main/java/org/spoken_tutorial/health/elasticsearch/JsonService/JsonService.java

```diff
@@ -83,4 +84,165 @@ public String saveNarrationToFile(String url, String documentId) throws ParseExc
         }
         return document;
     }
+
+    public String saveNarrationAndCuefScriptoHtmlFile(String url, int tutorialId, int lanId)
+            throws ParseException, IOException {
+
+        String jsonString = "";
+        boolean flag = true;
+        jsonString = restTemplate.getForObject(url, String.class);
+
+        String document = "";
+
+        Path jsonDir = Paths.get(mediaRoot, Config.uploadDirectoryScriptJsonFile);
+        Files.createDirectories(jsonDir);
+        Path jsonfilePath = Paths.get(mediaRoot, Config.uploadDirectoryScriptJsonFile, tutorialId + ".json");
+        File jsonfile = jsonfilePath.toFile();
+        if (jsonfile.exists()) {
```

Comment:
```suggestion
        if (Files.exists(jsonfilePath)) {
```
See if this works. Avoid path to file conversion.